bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -211,8 +213,19 @@ export default function Whiteboard(props) {
     const invalidShapeType = Object.keys(changedShapes)
       .find(id => !isValidShapeType(changedShapes[id]));
 
-    if (invalidChange || invalidShapeType) {
-      notifyNotAllowedChange(intl);
+    const shapeNumberExceeded = Object.keys(shapes).length > maxNumberOfAnnotations;
+    const isInserting = Object.keys(changedShapes)
+      .filter(
+        shape => typeof changedShapes[shape] === 'object'
+          && changedShapes[shape].type

```

Comment:
Changed shapes also contains the edited shapes, so currently it doesn't allow e.g. resizing a existing shape if the limit is reached.
Need to add a check if the shape didn't exist before.